src/utils/progressBar.ts

```diff
@@ -81,7 +88,13 @@ export namespace ProgressBar {
 
 export class DeployProgress extends ProgressBar {
   public constructor(private deploy: MetadataApiDeploy) {
-    super();
+    super({
```

Comment:
Construct deploy status with an additional custom field that will display current deploy status